### DISCUSSION TITLE
Pairing Issue Possible Fix for 9CUS

### DISCUSSION THREAD

**Discussion by Porkenstien** [alienatedsec/yi-hack-v5#238] [Show and tell]
Howdy,
Camera Details-
Yi Outdoor Cam(circa 2017)
Firmware: 3.0.0.0D
Hardware ID: 9CUS
Yi-Hack-V5 Ver: 0.3.7beta

Ok, so after wrestling with repeated pairing(Pairing Timed Out) failures, and thinking I had tried everything, I came across a fix(for my cam anyway). During a visit to my firewall servers logs for a completely different reason not realated to my camera situation, I noticed that the SquidGuard Proxy had a HUGE amount of attempts(200k) to contact google-analytics.com over the 2 days I had been tinkering with my camera. At this moment it became clear that it was likely the Yi/Kami App that is needed to pair the camera that was generating the increased traffic failures. After disabling my SquidGuard Proxy temporarily, the camera and Yi app paired-up instantly. Which has now allowed my Yi Cam running @alienatedsec hack the way it should.

So the short answer, if you run an advanced Proxy server(as I do), it may be worth disabling the Proxy for an hour and try pairing the camera.

Hope this helps someone else:)

**Comment by alienatedsec** [comment]
#204 #136 #124 #139
